### ISSUE TITLE
Percent Tracker for Stronghold

### ISSUE THREAD

**Issue by Wingysam** [GalaxyRoblox/Issues-Public#50]
Add an element to the UI that shows you how you're doing in terms of getting a lootpool item from Kneall Stronghold.

**Comment by PataNautic** [comment]
@Wingysam I think damage% should just be a stat you can see for any ship that has drops from a loot pool. Maybe something toggleable?

**Comment by Wingysam** [comment]
I think we should have a boss UI at the top of the screen that shows the HP and breakpoints and your %.

**Comment by PataNautic** [comment]
> I think we should have a boss UI at the top of the screen that shows the HP and breakpoints and your %.

I was considering a UI element for \[redacted\]-- very similar to the Ommoran Heartstone from DRG
![image](https://github.com/user-attachments/assets/6942973f-e1dc-4236-9a12-ec0dddf85cc7)

Perhaps we can develop this for \[redacted\] this Fall and then repurpose it for Kneall Stronghold?

**Comment by Wingysam** [comment]
Yeah something like that looks nice! Btw this is in Issues-Public, so I removed the event leaks from your reply. It's fine that people know we're planning to have a health UI for the event this fall though!

**Comment by PataNautic** [comment]
whoops 

![frozone](https://github.com/user-attachments/assets/6467e303-c35c-47e5-a588-b1bf580b66ef)

**Comment by Wingysam** [comment]
Closed in favor of #199
